environments/hashistack/build/1_hashicorp_tools.sh

```diff
@@ -30,7 +30,7 @@ install_from_apt() {
     sudo echo "deb [arch=amd64] https://apt.releases.hashicorp.com ${CLI_REPO} main" \
         > /etc/apt/sources.list.d/hashicorp.list
     sudo apt-get update
-    sudo apt-get install -y terraform=0.13.3 vault=1.5.2 consul=1.8.4 nomad=0.12.4 packer=1.6.2
+    sudo apt-get install -y terraform=0.13.3 vault=1.5.2 consul=1.8.4 nomad=0.12.4 packer=1.6.2 waypoint=0.1.3 boundary=0.1.1
```

Comment:
Can I piggyback on this to update Vault version please?  Vault 1.5.5 was just released today. 

```suggestion
    sudo apt-get install -y terraform=0.13.3 vault=1.5.5 consul=1.8.4 nomad=0.12.4 packer=1.6.2 waypoint=0.1.3 boundary=0.1.1
```